test-cases/artifacts/azure-image.yaml

```diff
@@ -3,7 +3,7 @@ azure_region_name: "eastus"
 use_preinstalled_scylla: true
 backtrace_decoding: false
 azure_image_db: ''
-azure_instance_type_db: 'Standard_L8s_v2'
+azure_instance_type_db: 'Standard_L8s_v3'
```

Comment:
This should be something much smaller.
L2S_V3 is fine.

Reply:
this is the smallest

Reply:
ok, thx.